india_compliance/public/js/transaction.js

```diff
@@ -79,7 +79,13 @@ async function update_gst_details(frm, event) {
     const party_type = india_compliance.get_party_type(frm.doc.doctype).toLowerCase();
     const party_fieldname = frm.doc.doctype === "Quotation" ? "party_name" : party_type;
     const party = frm.doc[party_fieldname];
-    if (!party) return;
+
+    const is_outward_material_transfer_or_issue =
```

Comment:
more readable check I feel is:
same_gstin_stock_entry

Party or same_gstin_stock_entry should be there.